app-backend/db/src/test/scala/com/azavea/rf/database/PropTestHelpers.scala

```diff
@@ -529,4 +529,10 @@ trait PropTestHelpers {
       INSERT INTO task_sessions VALUES
         (uuid_generate_v4(), now(), now() - INTERVAL '10 min', NULL, ${fromStatus}, NULL, ${taskSessionCreate.sessionType},
         ${user.id}, ${taskId})""").update.withUniqueGeneratedKeys[UUID]("id")
+
+  def addClasses(
+      label: AnnotationLabelWithClasses.Create,
+      classes: List[UUID]
+  ): AnnotationLabelWithClasses.Create =
+    label.copy(annotationLabelClasses = classes.take(1))
```

Comment:
This is a nit, but I think this would be clearer taking only a single UUID -- as is, taking the `List`, it appears to advertise that it will add several classes. It would be pretty easy to miss this and build expectations around adding several classes, then get confused and spend a while tracking down what you did in the db wrong.

```suggestion
  def setLabelClass(
      label: AnnotationLabelWithClasses.Create,
      classToAdd: UUID
  ): AnnotationLabelWithClasses.Create =
    label.copy(annotationLabelClasses = List(classToAdd))
```